### DISCUSSION TITLE
PrimeNg v19 supported templates / roadmap

### DISCUSSION THREAD

**Discussion by damgel** [primefaces/community#3282] [PrimeNG Templates]
It would be nice if they would update the templates or at least the documentation so that everyone knows which templates are being migrated or supported in v19. We have a lot of v17 projects and currently using a few paid templates (Apollo, Verona), a public roadmap for the templates would be so helful to plan the migrations.

**Comment by rubenheymans** [comment]
so apollo won't work with primeng 19? I just bought it

**Reply by siva-geddada** [reply]
Even i too bought the Apollo template and waiting for v19 update due to primeng massive update in v18.

**Reply by theRhOussama** [reply]
same thing here ! any updates ?

**Comment by marcusengel** [comment]
Is there any template up-to-date/compatible with angular 19?
Looks like all templates are still on angular 17 and not even updated to angular 18. Not to mention v19?! Unfortunately before payment this info is not available. Only the free "Sakai" is marked as v17. And the version dropdown in the top right corner does nothing.

**Reply by damgel** [reply]
We're all in the same boat mate; waiting for an official reply/roadmap. 🤷🏿

**Comment by hduru** [comment]
Hello, my team and I are also waiting for a roadmap regarding this topic. We are using the Diamond template. Could we at least get an estimated timeframe?

**Comment by djay1977** [comment]
Hello,

Normally the roadmap for the template update was for the end of December 2024.
I guess it should happen soon, I'm also waiting for Ultima.

What I know, the migration of templates to TailwindCSS and Design Tokens has been done for PrimeVue.
The only templates that will be migrated are those present in PrimeVue. The others have been removed.

https://primevue.org/templates/

**Comment by cagataycivici** [comment]
Please view the [roadmap](https://primeng.org/roadmap) to track the progress. We're adding PrimeNG v19 support, styling with the new theming, ngmodule to standalone, primeflex to tailwind and extreme QA flow, so taking a bit more time than initially expected due to scope  being expanded.

[Diamond](https://diamond.primevue.org) and [Poseidon](https://poseidon.primevue.org) will have redesigns as well, see the Vue versions for a preview.
